meta-balena-common/recipes-core/healthdog/healthdog_0.1.0.bb

```diff
@@ -1,52 +1,58 @@
-# Auto-Generated by cargo-bitbake 0.3.7
+# Auto-Generated by cargo-bitbake 0.3.12
 #
 inherit cargo
 
 # If this is git based prefer versioned ones if they exist
 # DEFAULT_PREFERENCE = "-1"
 
 # how to get healthdog could be as easy as but default to a git checkout:
-# SRC_URI += "crate://crates.io/healthdog/0.1.0"
-SRC_URI += "git://git@github.com/resin-os/healthdog-rs.git;protocol=ssh"
-SRCREV = "a10155844f251d00ec49348e58b2846803386085"
+# SRC_URI += "crate://crates.io/healthdog/1.0.0"
+SRC_URI += "git://github.com/balena-os/healthdog-rs.git;protocol=https;nobranch=1"
+SRCREV = "v1.0.0"
```

Comment:
SRCREV really should only contain sha1s. I would rename the recipe to healthdog_1.0.0.bb and use a sha1 here.

Reply:
@alexgg :+1: noted, thanks.